staticcheck.conf

```diff
@@ -0,0 +1,3 @@
+# Checks being ignored:
```

Comment:
Why is this required?

Reply:
We wanted to see if adding a `staticcheck.conf` with this rule would silence the Coding standards violations, but it looks like Tics isn't picking this up so our Coding standards TQI score remains the same. See https://github.com/kadinsayani/microcloud/actions/runs/11446739960/job/31905615699.

Reply:
We can still keep this staticcheck rule since it doesn't apply to us.

Reply:
Thanks!